warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -178,15 +189,40 @@ public Object visit(ASTOrNode node, Object data) {
             else {
                 // recurse on the eq children in this subset
                 copyChildren(eqNodes, children, data);
+                track(data, field, eqNodes.size() - 1);
                 
                 // recurse on the range children in this subset
                 copyChildren(rangeNodes, children, data);
+                
+                int numBatches = (int) Math.ceil(rangeNodes.size() / (double) Math.max(1, config.getMaxRangesPerRangeIvarator()));
+                numBatches = Math.min(Math.max(1, config.getMaxOrRangeIvarators()), numBatches);
+                track(data, field, rangeNodes.size() - numBatches);
             }
         }
         
         return children(newNode, children.toArray(new JexlNode[children.size()]));
     }
     
+    /**
+     * If data is set update it for the given field to include the new possible reduction, otherwise do nothing
+     * 
+     * @param data
+     * @param field
+     * @param reduction
```

Comment:
Define reduction -- I assume it's the number of nodes eliminated from the query via pushdown?